spring-cloud-skipper-server/src/test/java/org/springframework/cloud/skipper/repository/PackageMetadataMvcTests.java

```diff
@@ -59,9 +64,28 @@ public void testProjection() throws Exception {
 				.andExpect(jsonPath("$._embedded.packageMetadata[0].iconUrl")
 						.value("http://www.gilligansisle.com/images/a2.gif"))
 				.andExpect(jsonPath("$._embedded.packageMetadata[0].description").doesNotExist())
+				.andExpect(jsonPath("$._embedded.packageMetadata[0]._links.install.href")
+						.value("http://localhost/packageMetadata/3/install"))
```

Comment:
Once this is addressed, we may not need `deleteAllBeforeTests` and type annotations `Transactional` and ` the `FixMethodOrder ` as well.